speech/google/cloud/speech/_gax.py

```diff
@@ -104,9 +104,9 @@ def async_recognize(self, sample, language_code=None,
         audio = RecognitionAudio(content=sample.content,
                                  uri=sample.source_uri)
         api = self._gapic_api
-        response = api.async_recognize(config=config, audio=audio)
+        operation_future = api.async_recognize(config=config, audio=audio)
 
-        return Operation.from_pb(response, self)
+        return Operation.from_pb(operation_future.last_operation_data(), self)
```

Comment:
Okay, I see the problem. I will talk to @bjwatson about this tomorrow. I _really_ want this to return an `_OperationFuture`, which means the HTTP surface needs to return something with a similar interface.